tests/unit/src/test/scala/tests/OnTypeFormattingSuite.scala

```diff
@@ -37,6 +37,40 @@ class OnTypeFormattingSuite extends BaseLspSuite("onTypeFormatting") {
        |}""".stripMargin
   )
 
+  check(
```

Comment:
Could we add similar test to range formatting suite? With different indents?

Reply:
Sure, added.